modules/sw360/sw360-client/src/main/java/org/eclipse/sw360/antenna/sw360/client/rest/resource/releases/SW360Release.java

```diff
@@ -291,6 +294,17 @@ public SW360Release setProprietary(boolean proprietary) {
         return this;
     }
 
+    @JsonIgnore
+    public String getHomepageurl() {
```

Comment:
One minor thing: Should the name of the property be homepageUrl rather than homepageurl? Camel case is probably more readable.